doc/mgr/dashboard.rst

```diff
@@ -156,7 +156,7 @@ command.
 The credentials of an existing user can also be obtained by using
 `radosgw-admin`::
 
-  $ radosgw-admin user info --uid=<user id>
+  $ radosgw-admin user info --uid=<user_id>
 
 Finally, provide the credentials to the dashboard module::
 
```

Comment:
Please add `$ ceph dashboard set-rgw-api-secret-key <user id>` to the section below ⬇⬇⬇